apps/website/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@sentry/browser": "5.15.5",
     "@sentry/integrations": "5.15.5",
+    "@clr/core": "^4.0.0",
```

Comment:
Why was this one set to 4.0 instead of latest like the others?

Reply:
I tried something with it but forgot to turn in back, thanks for seeing it!